content/zh/docs/reference/config-api/client-authentication.v1beta1.md

```diff
@@ -60,95 +77,135 @@ should use to contact the API.</td>
 
 
 
-**Appears in:**
+<!--**Appears in:**-->
+**出现在：**
 
 - [ExecCredentialSpec](#client-authentication-k8s-io-v1beta1-ExecCredentialSpec)
 
 
+<!--
 Cluster contains information to allow an exec plugin to communicate
 with the kubernetes cluster being authenticated to.
 
 To ensure that this struct contains everything someone would need to communicate
 with a kubernetes cluster (just like they would via a kubeconfig), the fields
 should shadow "k8s.io/client-go/tools/clientcmd/api/v1".Cluster, with the exception
 of CertificateAuthority, since CA data will always be passed to the plugin as bytes.
+-->
+Cluster 中包含允许 exec 插件与 Kubernetes 集群进行通信身份认证时所需
```

Comment:
```suggestion
Cluster 中包含允许 exec 插件与 Kubernetes 集群进行通信身份认证时所需
```
Referring to other Chinese localizations, the proper noun `cluster` is usually translated

Reply:
the Cluster is a Go Struct，so I leave it a lone

Reply:
thank,I get it